lucuma-schemas/src/clue/scala/lucuma/schemas/ObservationDB.scala

```diff
@@ -53,6 +54,8 @@ trait ObservationDB {
     type PosLong                    = eu.timepit.refined.types.numeric.PosLong
     type PosShort                   = eu.timepit.refined.types.numeric.PosShort
     // Core Types
+    type ProgramReference           = String           // TODO: Switch to core type when it is actually in core
```

Comment:
Is this planned to go into core? In the meantime, maybe we can add a `NewType` in this project.

Reply:
Yeah. That is currently only private in ODB, but will be moved to core. Since we're not actually using it right now, I thought we could just punt for a bit.